inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -1939,6 +1940,15 @@ void convertFunctionToICNNNetwork(const std::shared_ptr<const ::ngraph::Function
             cnnLayer->params[ExecGraphInfoSerialization::ORIGINAL_NAMES] = originalNames;
         }
 
+        if (auto subgraph = ::ngraph::as_type_ptr<ngraph::op::Subgraph>(layer)) {
```

Comment:
we already have the utility operation SubGraph in the nGraph. Could you try to use it instead of introducing new operations?

Reply:
I would prefer to stick to this simple class but rename it. BTW, `ngraph::pass::low_precision::Subgraph` is another subgraph